mmdeploy/codebase/base/backend_model.py

```diff
@@ -35,15 +35,20 @@ def __init__(self,
     def _build_wrapper(backend: Backend,
                        backend_files: Sequence[str],
                        device: str,
+                       input_names: Optional[Sequence[str]] = None,
```

Comment:
could check if this brings BC breaking.

Reply:
No BC is involved here I guess as the default value `None` does not influence anything for other backends.